commerce/endpoint/cart/CartItemCollectionExtensionSample.cls

```diff
@@ -0,0 +1,45 @@
+/**
+ * Sample extension for updating the quantity of a cart item while adding an item to the cart.
+ * This corresponds to the endpoint /commerce/webstores/${webstoreId}/carts/${activeCartOrId}/cart-items
+ * and is identified by the EPN Commerce_Endpoint_Cart_ItemCollection for registration/mapping.
+ */
+public with sharing class CartItemCollectionExtensionSample extends ConnectApi.BaseEndpointExtension {
```

Comment:
We have tested these samples in our local org and org farm.

Reply:
Please add the Apex tests, we need 75% code coverage on every Apex file being added?